app/assets/stylesheets/modules/_tables.styl

```diff
@@ -20,6 +20,7 @@
 .table
   width 100%
   border-collapse separate
+  margin 2% 0
```

Comment:
@yunjuc can you explain more about what this is for, and perhaps illustrate it?

Reply:
It adds 2% margin on the top and bottom of a table, and 0 margin on the right and left. Therefore, the spaces between a table and the elements above and after it become wider.

Reply:
This is not necessary for fixing the training notification issue, right?

I browsed a little bit, and in many cases I think this extra space spreads things out too much. It's worthwhile to consider what global layout adjustments might improve usability (and maybe my first impression of this one is off base), but I'd like to have a clear understanding of the rationale, and something like that should be done in a separate PR.

Reply:
Yes, this is one separate from the notification issue and I should do it on sperate PR.

The original thought of the fix is I saw the campaign text is right after the previous table and I want to give it some breathing space. It's enclosed in section tag so I first added the margin on the section (see dom samples below). It caused the unexpected space because the purple welcome section use a background color and it's only seen when you're not logged in, so I missed that. 
<body>
   ....
   <section>
     <table></table>
  </section>
   ...  
</body>

Later I changed it to add margin to the table tag. I can do it either only on this specific table vs. a global change on all table. The reason I did it globally is to make the style consistent, so when a table is created, it has the same spacing as other tables. And the trade-off is the added space you see, and I think some extra space to make each element more distinct from each other is not a bad thing, as long as it's not an awkward gap. Thoughts?

On a side note, I'm having a `ERR_TOO_MANY_REDIRECTS` issue and can't open localhost these two days. There are a lot of rollback and 302 redirect on guard, and I tried to clear the cookies but that doesn't help.

Reply:
No, I've never seen that before on localhost. Usually when I've run into that in other places, it's because of the browser caching bad 302 routes. The way to clear redirect caches varies by browser, but try it's worth a try. As a first step, see if the works properly in a different browser; if so, it's probably cached redirects.

Reply:
I'll try to resolve this. Thanks.